scalafix-core/shared/src/main/scala/scalafix/patch/Patch.scala

```diff
@@ -48,12 +48,14 @@ sealed abstract class Patch {
   def ++(other: Iterable[Patch]): Patch = other.foldLeft(this)(_ + _)
   def isEmpty: Boolean = this == EmptyPatch
   def nonEmpty: Boolean = !isEmpty
+  def atomic: Patch = AtomicPatch(this)
 }
 
 //////////////////////////////
 // Low-level patches
 //////////////////////////////
 trait LowLevelPatch
+
```

Comment:
this fucks up sublime syntax highlighting.

Reply:
awww, maybe get a better editor 😜